core-addon/FirefoxPrivilegedApi.js

```diff
@@ -4,17 +4,24 @@
 
 "use strict";
 
-const { TelemetryController } = ChromeUtils.import(
-  "resource://gre/modules/TelemetryController.jsm"
-);
 const { XPCOMUtils } = ChromeUtils.import(
   "resource://gre/modules/XPCOMUtils.jsm"
 );
 
+const { RemoteSettings } = ChromeUtils.import(
+  "resource://services-settings/remote-settings.js"
+);
+
+const { TelemetryController } = ChromeUtils.import(
+  "resource://gre/modules/TelemetryController.jsm"
+);
+
```

Comment:
Why was this moved? (Was linting complaining or something?) - Nothing to do, but I'm curious :)

Reply:
Just putting them in lexical order :) I wonder if there's an eslint rule for that.

Reply:
I guess by that logic `XPCOMUtils` should be moved below but it's a common practice to have it be the first so it can be used for lazy imports (which I will likely switch these to)

Reply:
Makes sense!